config/admin.config.php

```diff
@@ -131,6 +156,12 @@
             'collection_query_whitelist' => array('version'),
         ),
     ),
+    'zf-rpc' => array(
+        'ZF\Apigility\Doctrine\Admin\Controller\DbAutodiscovery' => array(
```

Comment:
sorry @TomHAnderson It shouldn't have ended here, just corrected the array key for "DoctrineAutodiscovery", which is a controller :)